consumer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM docker.arvancloud.ir/python:3.11
+
+RUN pip3 install kafka-python six pymongo
```

Comment:
find a solution to add dependecy manegement like version tracking. do not call pip install packages like this.

Reply:
fixed